pkg/dyninst/testdata/decoded/sample/testMassiveString.json

```diff
@@ -0,0 +1,40 @@
+[
+  {
+    "service": "sample",
+    "ddsource": "dd_debugger",
+    "logger": {
+      "name": "",
+      "method": "main.testMassiveString",
+      "version": 0,
+      "thread_id": 0,
+      "thread_name": ""
+    },
+    "debugger": {
+      "snapshot": {
+        "id": "[id]",
+        "timestamp": "[ts]",
+        "language": "go",
+        "stack": "[stack-unredact-me]",
+        "probe": {
+          "id": "testMassiveString",
+          "location": {
+            "method": "main.testMassiveString"
+          }
+        },
+        "captures": {
+          "entry": {
+            "arguments": {
+              "x": {
+                "type": "string",
+                "size": 126417,
+                "notCapturedReason": "pruned",
```

Comment:
I think the protocol for strings is `"truncated": true` not `"notCapturedReason"`, right?